dramatiq/__init__.pyi

```diff
@@ -0,0 +1,31 @@
+from .actor import Actor as Actor
```

Comment:
Are the aliases required for some reason?

Reply:
Yes, `mypy` requires aliases to be configured this way. It is related to `implicit_reexport` setting. And sometimes `mypy` does not tend to see these values with `as` part.

Reply:
Got it. Thank you!

Reply:
I think mypy should also be fine with it if you provide an explicit `__all__` and include the imported object there.